From 88b03288a94034f459c496ccd00babc7c0dca0a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Mon, 18 Oct 2021 03:58:56 +0200
Subject: [PATCH 292/515] arm64: dts: rk3399-pinephone-pro: Power off goodix
 touchscreen in sleep

Otherwise it consumes 200mW of power when touched during system sleep.
It can't wake up the phone anyway.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f8fab58259c0..2e1eeb616e46 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -602,6 +602,7 @@ touchscreen@14 {
 		VDDIO-supply = <&vcc3v0_touch>;
 		touchscreen-size-x = <720>;
 		touchscreen-size-y = <1440>;
+		poweroff-in-suspend;
 	};
 
 	light-sensor@48 {
-- 
2.35.3

